usaspending_api/etl/award_helpers.py

```diff
@@ -320,6 +320,7 @@ def prune_empty_awards(award_tuple: Optional[tuple] = None) -> int:
       UPDATE financial_accounts_by_awards
         SET
           update_date = now(),
+          linked_date = now(),
```

Comment:
Should this be `now()` or `null` ?

Reply:
If we set this to `null`, there will be no way to indicate to the ElasticSearch loader that this should be reloaded.

Reply:
What's the scenario covered here?

i.e. why would this be called / why is the award_id being set to null?

If because the award is being deleted, then that will be covered by the delete-tracking of awards already baked into the nightly delta (I believe such a thing exists, but haven't looked for it).

Since the nighty delta goes after the award records first, if the award is now null, it won't be able to figure out _which_ **award** data in elasticsearch needs to be changed. While a `linked_date` indicates that faba record was changed, we can't discover what award was affected.